### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by sampler36**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [x] I have commented my code, particularly in hard-to-understand areas

- [x] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -1,25 +0,0 @@
-import React from 'react';
```
Comment (ConConRob) [inline src/components/NavBar.js]: Why are these files deleted

File: package.json [hunk 2]
```diff
@@ -3,7 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.3",
```
Comment (ConConRob) [inline package.json]: I think you are still working from a out of date branch.

File: package.json [hunk 3]
```diff
@@ -3,7 +3,10 @@
...
     "axios": "^0.18.0",
+    "knex": "^0.16.5",
```
Comment (ConConRob) [inline package.json]: We are not using a SQL DB anywhere in the project

File: package.json [hunk 4]
```diff
@@ -3,7 +3,10 @@
...
+    "material-ui": "^0.20.2",
```
Comment (ConConRob) [inline package.json]: Why do we have this?

File: package.json [hunk 5]
```diff
@@ -13,7 +16,8 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "socket.io-client": "^2.2.0",
-    "styled-components": "^4.2.0"
+    "styled-components": "^4.2.0",
+    "uid": "^0.0.2"
```
Comment (ConConRob) [inline package.json]: What is this? If it is a ID generator, why do we need it here?

File: src/components/layout/navbar/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  return(
+    <nav className="nav-wrapper navy darken-2">
+       <div className="container">
+       <NavLink to="/" className="brand-logo">
+       <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" /> </header>
+       </NavLink>
+       <LoggedIn />
+       <LoggedOut />
+        {/* {navlinks} */}
```
Comment (Pav0l) [inline src/components/layout/navbar/NavBar.js]: I think these need conditional rendering based on whether the user is logged in or not

Comment (sampler36) [reply src/components/layout/navbar/NavBar.js]: Done

File: src/App.js [hunk 7]
```diff
@@ -71,10 +73,11 @@ class App extends React.Component {
       dispatchFetchAllUsers,
       dispatchLogout,
     } = this.props;
-
+console.log('gththt')
+ console.log(this.props)
```
Comment (Pav0l) [inline src/App.js]: these `console.log`s should not be here

File: src/components/layout/navbar/NavBar.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+// NavBar.propTypes = {
+//   loggedIn: PropTypes.bool.isRequired,
+// };
+const mapStateToProps = (state) => {
+  return{
+    // auth: state.mongodb.auth
+  }
+}
+
+export default connect(mapStateToProps)(NavBar);
```
[the remark this answers is not in the document]
Comment (Pav0l) [reply src/components/layout/navbar/NavBar.js]: this is still not resolved

[the remark this answers is not in the document]
Comment (Pav0l) [reply src/components/layout/navbar/NavBar.js]: To export the file simply:
`export default Navbar;`
should be enough.

`mapStateToProps` and `connect` can be deleted

File: src/components/layout/navbar/SignedInLink.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { ADMIN,SUPER_ADMIN, RECEPTIONIST  } from '../../../utils/userTypes'
+
+
+
+const LoggedIn = ({userType}) => {
+  console.log(userType)
```
Comment (Pav0l) [inline src/components/layout/navbar/SignedInLink.js]: do we need this `console.log`?

Comment (sampler36) [reply src/components/layout/navbar/SignedInLink.js]: no i forgot to remove it

File: src/App.js (hunk 7 above)
Comment (sampler36) [reply src/App.js]: removed

File: src/components/layout/navbar/NavBar.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (sampler36) [reply src/components/layout/navbar/NavBar.js]: deleted connect

File: package.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (sampler36) [reply package.json]: deleted

Comment (sampler36) [reply package.json]: deleteed

File: package.json (hunk 4 above)
Comment (sampler36) [reply package.json]: deleted
